Logging/ARKLogMessage.m

```diff
@@ -19,7 +19,6 @@
 //
 
 #import "ARKLogMessage.h"
-
```

Comment:
nit: please add the newline back. It's Square style to have your own `.h` import be separated by a newline from other class imports.